### PR TITLE
Deploy the web editor to kubernetes

### PR DESCRIPTION
**Pull request by sandhose**
This sets up:

 - Jib to build the container
 - Kustomize/kubernetes manifests to deploy the app in the cluster
 - Skaffold to automate the tagging and deployment of the app
 - GitLab-CI for continuous delivery

Right now it is deployed to http://hypertest.kube.zazuko.com and deploys on every commit on master.

### CODE REVIEW

**Comment by sandhose** [comment]
Travis breaks because I had to add Jib to the default `package` target because of Skaffold. I might ditch Skaffold to avoid that issue.

File: app.hypermedia.testing.dsl.cli/pom.xml [hunk 1]
```diff
@@ -12,6 +12,12 @@
         <maven.compiler.target>1.8</maven.compiler.target>
     </properties>
 
+    <parent>
+        <groupId>app.hypermedia.testing.dsl</groupId>
+        <artifactId>app.hypermedia.testing.dsl.parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
```
Comment (tpluscode) [inline app.hypermedia.testing.dsl.cli/pom.xml]: Why is this needed?

Comment (sandhose) [reply app.hypermedia.testing.dsl.cli/pom.xml]: This is set in every other module but this one for some reason. Without this I had an error saying it didn't found the `jib:build` target in this module

Comment (tpluscode) [reply app.hypermedia.testing.dsl.cli/pom.xml]: > for some reason

Hah, quite simple. The others were built from generator and this I half-handcrafted :)
